apps/wallets/smart-wallet/react/src/app/App.tsx

```diff
@@ -14,4 +15,4 @@ function App() {
     );
 }
 
-export default App;
+export default withProviders(App);
```

Comment:
Overall the PR looks good. I'm not certain though that we should use a providers HOC here. I'm wondering if we should add them separately like how we're adding the <AppProvider> :thinking: 

I'm not sure what kind of implication that will have on your changes though. What do you think?

Reply:
My reasoning was: I wanted a way to inject multiple auth providers, and wrapping the App around custom Provider components seemed to be the pattern for most of these React sdks.
I don't understand much of the implications tho!

Reply:
Totally fair and that makes sense. I'm not apposed to this I'm just concerned it's more complex than it needs to be. Lemme pull it in on my end and see what I can do.

Reply:
posting my commit here for visibility: https://github.com/Crossmint/crossmint-sdk/pull/662/commits/b2e1484356e2b9fd60539a6734a8a03cd2bf9455